Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Clickhouse/DriverFee.hs

```diff
@@ -58,12 +61,13 @@ driverFeeTTable =
       specialZoneAmount = "special_zone_amount",
       govtCharges = "govt_charges",
       collectedAt = "collected_at",
-      collectedBy = "collected_by"
+      collectedBy = "collected_by",
+      updatedAt = "updated_at"
     }
 
 type DriverFee = DriverFeeT Identity
 
-$(TH.mkClickhouseInstances ''DriverFeeT 'SELECT_FINAL_MODIFIER)
+$(TH.mkClickhouseInstances ''DriverFeeT 'NO_SELECT_MODIFIER) -- 'SELECT_FINAL_MODIFIER
```

Comment:
we should remove `FINAL` modifier, right @0utkarsh ?

Reply:
Yes, should be removed